ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+### Steps to Reproduce
+
+1. 
+2. 
+3. 
+
+### Expected Behavior
+
+### Actual Behavior
+
+### Environment
+
+<!--
+1. From the Visual Studio main menu, select: Help > Visual Studio > Show Details > Copy Info (https://developer.xamarin.com/guides/cross-platform/troubleshooting/questions/version-logs/)
+
+2. Paste below this comment block.
+-->
```

Comment:
I do not see this menu item (VSM). Do you mean _Visual Studio → About Visual Studio → Show Details → Copy Information_?

You might also want to add an empty code block below into which the environment info is pasted. It will probably render more legibly because the formatting of what is copied is white-space indented:

````markdown
<!--
...
2. Paste into the code block below (between ```)
-->

```
```

````

### See the difference

```
=== Visual Studio Community 2017 for Mac (Preview) ===

Version 7.3 Preview (7.3 build 764)
Installation UUID: 2701a58e-0630-4e42-b81d-229161066215
Runtime:
	Mono 5.4.1.7 (2017-06/e66d9abbb27) (64-bit)
	GTK+ 2.24.23 (Raleigh theme)

	Package version: 504010007

...
```

### Versus

=== Visual Studio Community 2017 for Mac (Preview) ===

Version 7.3 Preview (7.3 build 764)
Installation UUID: 2701a58e-0630-4e42-b81d-229161066215
Runtime:
	Mono 5.4.1.7 (2017-06/e66d9abbb27) (64-bit)
	GTK+ 2.24.23 (Raleigh theme)

	Package version: 504010007

...

Reply:
@chamons the new release notes templates have instructions for this - and also cover VS Windows.

Reply:
Hmm, I thought I copied that straight from our docs (after fixing the name). Let me fix that by using the template from release notes...